src/tests/rest-calls-200-404.spec.ts

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import { AxiosHarTracker } from '../axios-har-tracker'
+import { writeFileSync } from 'fs';
+
+describe('Check axios-har-tracker for status 200 and 404', () => {
```

Comment:
Why are there 3 separate spec files: 200 and 404, 302, 500. All this can be done in one spec, in separate `it` statements.

Reply:
discussed